templates/dataset.yaml

```diff
@@ -20,19 +20,55 @@ dataset:
     # convention. Any other variables in your dataset that cannot be mapped to the EFont variables will be
     # ignored and cannot be used for calculations at this time.
-    variable_name:
+     timestamps: timestamps
+     baseline_electric_power_profile: 
+        variable_name:
+            name: variable_name 
         building_id:
-            name: building_id
-        availability:
-            name: availability
-            filepath: filepath/to/variable_name # [OPTIONAL] Supply filepath to dataset where variable can be found if different from <dataset>:<filepath> value
-        timestamps: timestamps
-        baseline_electric_power_profile: baseline_electric_power_profile
-        baseline_electricity_consumption_profile: baseline_electricity_consumption_profile
-        baseline_natural_gas_consumption_profile: baseline_natural_gas_consumption_profile
-        flexible_electric_power_profile: flexible_electric_power_profile
-        flexible_electricity_consumption_profile: flexible_electricity_consumption_profile
-        flexible_natural_gas_consumption_profile: flexible_natural_gas_consumption_profile
-        load_profile_peak_timestamp: load_profile_peak_timestamp
-        load_profile_valley_timestamp: load_profile_valley_timestamp
-...
\ No newline at end of file
+            name: building_id 
+        filepath: filepath/to/variable_name # [OPTIONAL] Supply filepath to dataset where variable can be found if different from <dataset>:<filepath> value
+     baseline_electricity_consumption_profile: 
+        variable_name:
+            name: variable_name 
+        building_id:
+            name: building_id 
+        filepath: filepath/to/variable_name # [OPTIONAL] Supply filepath to dataset where variable can be found if different from <dataset>:<filepath> value  
+     baseline_natural_gas_consumption_profile: 
+        variable_name:
+            name: variable_name 
+        building_id:
+            name: building_id 
+        filepath: filepath/to/variable_name # [OPTIONAL] Supply filepath to dataset where variable can be found if different from <dataset>:<filepath> value  flexible_electric_power_profile: flexible_electric_power_profile
+     flexible_electricity_consumption_profile: 
+        variable_name:
+            name: variable_name 
+        building_id:
+            name: building_id 
+        filepath: filepath/to/variable_name # [OPTIONAL] Supply filepath to dataset where variable can be found if different from <dataset>:<filepath> value 
+     flexible_natural_gas_consumption_profile: 
+        variable_name:
+            name: variable_name
+        building_id:
+            name: building_id 
+        filepath: filepath/to/variable_name # [OPTIONAL] Supply filepath to dataset where variable can be found if different from <dataset>:<filepath> value 
+     
+     
+     # does this refer to operation states? 
```

Comment:
Yeah, `availability` is meant to refer to operation state. It was relevant when defining the `Power Payback Ratio` KPI. To be more specific maybe I should name it `operation_state` instead of `availability`?

Reply:
I would say so.. we can confirm that today..